src/collapse/perf.rs

```diff
@@ -1,4 +1,11 @@
+use addr2line;
+use memmap;
+use rustc_demangle;
```

Comment:
I don't think you need these any more in the 2018 edition?

Reply:
Cool, why doesn't the compiler warn about this?

Reply:
Good question. Seems like a bug. Should probably say "unnecessary `use`". My guess is that this is leftover from the old 2015 behavior.